### ISSUE TITLE
 Missing scheduler.step() in run_train

### ISSUE THREAD

**Issue by vaswojciech** [ybabakhin/nie-pdl-template-errors#11]
Missing scheduler.step() in run_train
